packages/core/android/build.gradle

```diff
@@ -181,10 +181,10 @@ dependencies {
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
   compileOnly "com.squareup.okhttp3:okhttp:3.12.13"
```

Comment:
I think we can switch to 4.x here actually, because in the end 4.x will be brought by the native SDK and this will be the version finally resolved by Gradle even if this SDK is integrated with old RN.

Reply:
I will do this in a further PR if that's ok, it is only used for the proxy so it's going to be a bit tedious to check everything still works after the update.

Reply:
Yes, it was a side note, so it is completely ok.

This dependency has `compileOnly`, so it is used only for the tests anyway, no need to check runtime behavior.